checador/app/views/layouts/application.html.erb

```diff
@@ -5,15 +5,15 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <link href="https://unpkg.com/ionicons@4.5.10-0/dist/css/ionicons.min.css" rel="stylesheet">
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
-    
+    <!--<p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>-->
```

Comment:
If you don't occupy it, delete it.

Reply:
It is recommended that you do not leave code in comments.

Remember that Git allows us to view this code on GitHub, in the commit in which it was created.

Reply:
I'm going to use it later, thank you, Yil